### ISSUE TITLE
Allow setting string value as radio field value

### ISSUE THREAD

**Issue by ksocha** [gettalong/hexapdf#116]
Lately I was experimenting with [acro_form.pdf](https://github.com/gettalong/hexapdf/files/5103985/acro_form.pdf) and programatically manipulating values of fields in the form.

It seems that it's impossible to do something like: 
```
field = doc.acro_form.field_by_name('Radio')
field&.field_value = 'button2'
```
because possible radio field values were defined as symbols. Is this valid?

I have multiple pdf docs with forms in them and I try manipulate form values using HexaPDF and rest API. It would be best if I could do the above without explicitly converting string to symbol.

**Comment by gettalong** [comment]
Yes, a radio button field value is a symbol because it references the key in an appearance dictionary, and PDF dictionary keys always have to be symbols. This mimics the way PDF works internally.

What is the problem with doing `value.intern` when assigning to a radio button field?

**Comment by ksocha** [comment]
Given that I modify multiple field of different types in a row, I would have to check filed type each time and do `value.intern` for radios only.

It would be much easier if line https://github.com/gettalong/hexapdf/blob/master/lib/hexapdf/type/acro_form/button_field.rb#L274 was changed to sth like:
```
elsif radio_button_values.include?(value.kind_of?(String)? value.intern : value)
```

Besides, I can create a widget without use of a symbol:
```
widget = rb.create_widget(pGE, value: "buttonValue", Rect: [200 + 50, 540, 240 + 50, 580])
```
Then `rb.radio_button_values` returns `['buttonValue']` so I guess using `value.intern` in this case would not work?
